### PR TITLE
doc/rados: line edit mon-lookup-dns top matter

### PR DESCRIPTION
**Pull request by zdover23**
Improve the syntax of the top matter of
doc/rados/operations/mon-lookup-dns.rst.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/rados/configuration/mon-lookup-dns.rst [hunk 1]
```diff
@@ -2,15 +2,19 @@
...
+Since Ceph version 11.0.0 (Kraken), RADOS has supported looking up monitors
+through DNS.
 
-This way daemons and clients do not require a *mon host* configuration directive in their ceph.conf configuration file.
+The addition of the ability to look up monitors through DNS means that daemons
+and clients do not require a *mon host* configuration directive in their
+``ceph.conf`` configuration file.
 
-Using DNS SRV TCP records clients are able to look up the monitors.
+By making use of a DNS update, clients and daemons can be made aware of changes
```
Comment (anthonyeleven) [inline doc/rados/configuration/mon-lookup-dns.rst:12]: Suggest 

With a DNS update

**Review by anthonyeleven — APPROVED** [review]
Suggestion

File: doc/rados/configuration/mon-lookup-dns.rst (hunk 1 above)
Comment (zdover23) [reply doc/rados/configuration/mon-lookup-dns.rst:12]: I'm chary of with-clauses, but if your fresh eyes tell you that it reads better, I'll take the benefit of those fresh eyes.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/50581 - Reef backport
https://github.com/ceph/ceph/pull/50582 - Quincy backport
https://github.com/ceph/ceph/pull/50583 - Pacific backport

**Pull request merged**